packages/cli/src/api/catalog.ts

```diff
@@ -227,26 +227,48 @@ export class Catalog {
     catalogs: Object,
     locale: string,
     key: string,
-    { fallbackLocale, sourceLocale }: GetTranslationsOptions
+    { fallbackLocales, sourceLocale }: GetTranslationsOptions
   ) {
     if (!catalogs[locale].hasOwnProperty(key)) {
       console.error(`Message with key ${key} is missing in locale ${locale}`)
     }
 
     const getTranslation = (locale) => catalogs[locale][key].translation
 
+    const getMultipleFallbacks = (locale) => {
+      let i = 0
+      const fL = fallbackLocales[locale]
+
+      // some probably the fallback will be undefined, so just search by locale
+      if (!fL) return null
+
+      if (Array.isArray(fL)) {
+        while(i <= fL.length) {
+          const fallbackLocale = fL[i]
+          if (catalogs[fallbackLocale]) {
+            return getTranslation(fallbackLocale)
+          }
+          i++
+        }
+      } else {
+        return getTranslation(fL)
+      }
+    }
+
     return (
       // Get translation in target locale
       getTranslation(locale) ||
-      // Get translation in fallbackLocale (if any)
-      (fallbackLocale && getTranslation(fallbackLocale)) ||
+      // Get translation in fallbackLocales.default (if any)
+      (fallbackLocales.default && getTranslation(fallbackLocales.default)) ||
+      // We search in fallbackLocales as dependent of each locale
+      getMultipleFallbacks(locale) ||
```

Comment:
shouldn't the `getMultipleFallbacks` has a higher precedence before `fallbackLocales.default`?

For example:

```json
{
  "fallbackLocales": {
    "en-gb": "en",
    "default": "cs"
  }
}
```

Then for a locale `en-gb` I should first try "en" and only if it doesn't exist I should fallback to "cs". What do you think?

Reply:
Absolutely